From: Guillaume Delacour <gui@iroqwa.org>
Subject: Don't call valgrind in tests
Last-Update: 2014-07-15

Index: sslh/t
===================================================================
--- sslh.orig/t
+++ sslh/t
@@ -58,8 +58,7 @@ for my $binary (@binaries) {
         my $user = (getpwuid $<)[0]; # Run under current username
         my $cmd = "./$binary -v -f -u $user --listen localhost:$sslh_port --ssh $ssh_address --ssl $ssl_address -P $pidfile";
         warn "$cmd\n";
-        #exec $cmd;
-        exec "valgrind --leak-check=full ./$binary -v -f -u $user --listen localhost:$sslh_port --ssh $ssh_address -ssl $ssl_address -P $pidfile";
+        exec $cmd;
         exit 0;
     }
     warn "spawned $sslh_pid\n";
